patches/ui-webui-resources-cr_components-theme_color_picker-theme_color_picker.html.ts.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/ui/webui/resources/cr_components/theme_color_picker/theme_color_picker.html.ts b/ui/webui/resources/cr_components/theme_color_picker/theme_color_picker.html.ts
+index c041589e048eba5c3dc2595ddf9a076a52822f5d..ac2e97c82ccc0903c93130a40a04af21b30357f5 100644
+--- a/ui/webui/resources/cr_components/theme_color_picker/theme_color_picker.html.ts
++++ b/ui/webui/resources/cr_components/theme_color_picker/theme_color_picker.html.ts
+@@ -27,21 +27,6 @@ export function getHtml(this: ThemeColorPickerElement) {
+       @click="${this.onDefaultColorClick_}"
+       basic-color>
+   </cr-theme-color>
+-  <cr-theme-color
```

Comment:
I guess we need to hurry up and write some Lit overriding methods - they seem to be converting more & more.

:cry:

Reply:
When we have `LitElement` overriding API, we could remove all this PR's change.
Filed https://github.com/brave/brave-browser/issues/38252